pkg/routing/localrouter.go

```diff
@@ -172,39 +234,82 @@ func (r *LocalRouter) OnRTCMessage(callback RTCMessageCallback) {
 	r.onRTCMessage = callback
 }
 
+func (r *LocalRouter) RegisterRoomService(svc rpc.RoomServerImpl) (err error) {
```

Comment:
it'd be great to take in context.Context as first param, so that it'd be easier to pass additional values across for Cloud

Reply:
we receive the remote context in handlers - it's not committed but i added this to make auth session passing work for cloud based on the grpc metadata feature